Makefile

```diff
@@ -255,6 +243,14 @@ YAML_FILES ?= $(shell find -name ngeo -prune -or \( -name "*.yml" -or -name "*.y
 yamllint: $(YAML_FILES)
 	yamllint --strict --config-file=yamllint.yaml -s $(YAML_FILES)
 
+.PHONY: greplint
```

Comment:
Please, what is this goal for ?

Reply:
When run the CI We use a project named testgeomapfish and it's easy to use it in the scaffold to make the CI pass...

Reply:
If I understand, this check there the dev does not have forgotten some 'testgeomapfish' hardly coded in scaffold.
I would add it as a comment.

Reply:
If so, why not rename the goal, e.g. verify_no_testgeomapfish_substitutions_left_in_scaffold ?

Reply:
Exactly @arnaud-morvan

Reply:
Relay too long, And I put a general name to be able to add some other rules of the came style...

Reply:
reading a ci ouptut for a failing build:
make greplint failed -> I don't understand, have to find the goal in makefile.
make verify_no_testgeomapfish_substitutions_left_in_scaffold failed -> I have a chance to imagine what happened.

reading a jenkis or travis or docker receipt chaning builds goals, I immediatly understand what is done at this step if not make greplint.

Reply:
I just add a message =>
```
...
ERROR: You still have a testgeomapfish in one of your scaffolds
geoportal/c2cgeoportal_geoportal/scaffolds/__init__.py:testgeomapfish
Makefile:248 : la recette pour la cible « additionallint » a échouée
make: *** [additionallint] Erreur 1
```